backend/src/main/java/com/jobwait/persistence/Util.java

```diff
@@ -0,0 +1,44 @@
+package com.jobwait.persistence;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.jobwait.persistence.adapters.AdapterException;
+import com.jobwait.persistence.adapters.PersistenceAdapter;
+
+public class Util {
```

Comment:
should definitely be renamed if you want to keep this in persistence folder, i am sure you already saw my last comment on renaming.

Reply:
yep, donezo